books/architecture/src/SUMMARY.md

```diff
@@ -1,3 +1,124 @@
 # Summary
 
-- [TODO](todo.md)
+[Cuprate Architecture](cuprate-architecture.md)
+[🟡 Foreword](foreword.md)
+
+---
+
+- [🟠 Intro](intro/intro.md)
+	- [🟡 Who this book is for](intro/who-this-book-is-for.md)
```

Comment:
The indenting in this page is non-standard, intentional?

Reply:
Do you mean the tabs? `rustfmt` doesn't do markdown files I guess. Changed to 4 spaces.